### PR TITLE
T766

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

File: src/core/page.js [hunk 1]
```diff
@@ -55,33 +55,46 @@ define( [ "core/logger", "core/eventmanager" ], function( Logger, EventManager )
...
+      }
+      node.parentNode.replaceChild( newNode, node );
 
       toClean = body.querySelectorAll( "*[butter-clean=\"true\"]" );
       for ( i = 0, l = toClean.length; i < l; ++i ) {
         node = toClean[ i ];
+
         node.removeAttribute( "butter-clean" );
-        node.removeAttribute( "data-butter" );
+        node.removeAttribute( "`er" );
```
Comment (jbuck) [inline src/core/page.js]: that doesn't look right

File: src/core/page.js [hunk 2]
```diff
@@ -55,33 +55,46 @@ define( [ "core/logger", "core/eventmanager" ], function( Logger, EventManager )
...
     this.getHTML = function( popcornStrings ){
-      var html, head, body, i, toClean, toExclude, node, base;
+      var html, head, body, i, toClean, toExclude, node, newNode, base, mediaElements;
 
       //html tag to which body and head are appended below
       html = document.createElement( "html" );
 
       // if there is already a snapshot, clone it instead of cloning the current dom
       if( !_snapshot ){
-        head = document.getElementsByTagName( "head" )[ 0 ].cloneNode( true );
```
Comment (jbuck) [inline src/core/page.js:65]: Any reason you want to get rid of the head cloning here? What if the template is doing something like... loading jquery?

File: src/core/page.js [hunk 3]
```diff
@@ -55,33 +55,46 @@ define( [ "core/logger", "core/eventmanager" ], function( Logger, EventManager )
...
         body = document.getElementsByTagName( "body" )[ 0 ].cloneNode( true );
       }
-      else{
-        head = _snapshot.head.cloneNode( true );
+      else {
         body = _snapshot.body.cloneNode( true );
       }
+      
+      head = document.getElementsByTagName( "head" )[ 0 ].cloneNode( true );
```
Comment (jbuck) [inline src/core/page.js:71]: Oh, you clone it down here. Whoops, sorry!

File: src/core/page.js (hunk 1 above)
Comment (secretrobotron) [inline src/core/page.js]: Hahahahh I don't even know... what happened.

**Pull request merged**